ts/components/ButtonDefaultOpacity.tsx

```diff
@@ -12,25 +13,39 @@ import { calculateSlop } from "./core/accessibility";
 const defaultActiveOpacity = 1.0;
 
 type CustomProps = {
+  hasFullHitSlop?: boolean;
   onPressWithGestureHandler?: true;
 };
 type Props = NativeBase.Button & React.Props<Button> & CustomProps;
 
-const smallSlop = calculateSlop(customVariables.btnSmallHeight);
-const xsmallSlop = calculateSlop(customVariables.btnXSmallHeight);
-const defaultSlop = calculateSlop(customVariables.btnHeight);
+const SlopsBySize = {
```

Comment:
```suggestion
const slopsBySize: Record<"small" | "xsmall" | "default", number>
```
as coding style, we use camel case to define a const that has no hard-coded

Reply:
No hard-coded _values_? :)

Reply:
yes, this is an example

```typescript
const withNoHardCodedValue = {small : calculateSlop(customVariables.btnSmallHeight)}
const WITH_HARCODED_VALUE = 5
```

Reply:
Thanks